wdl2cwl/tests/cwl_files/smoove.cwl

```diff
@@ -25,6 +27,10 @@ requirements:
             --fasta $(inputs.referenceFasta.path) \
             $(inputs.bamFile.path)
   - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: 15360
+  - class: ToolTimeLimit
+    timelimit: $(inputs.timeMinutes* 60)
```

Comment:
`timeMinutes` is missing from the inputs?

Reply:
This was the one that was discussed yesterday for which you suggested to come up with a javascript function.

Reply:
I don't follow. It is available in the inputs section with a default value.

Reply:
`timeMinutes` only occurs once in the generated CWL file that you checked in: https://github.com/common-workflow-lab/wdl-cwl-translator/blob/bed19b13188fb20b918b90b52eaf99598a20f101/wdl2cwl/tests/cwl_files/smoove.cwl

Reply:
oh yes. It's available in the inputs section as a bound input, which are addressed in the other PR #22 and not in the PR.

Reply:
That makes more sense 👍🏻. I've merged that PR, so please update this branch with the latest from `main`.